docs/src/solvers/cma_es.md

```diff
@@ -0,0 +1,38 @@
+# Covariance matrix adaptation evolutionary strategy
+
+```@meta
+CurrentModule = Manopt
+```
+
+The CMA-ES algorithm has been implemented based on [Hansen:2023](@cite) paper with basic Riemannian adaptations, related to transport of covariance matrix and its update vectors. Other attempts at adapting CMA-ES to Riemannian optimzation include [ColuttoFruhaufFuchsScherzer:2010](@cite).
+
+Covariance matrix transport between consecutive mean points is handled by `eigenvector_transport!` function which is based on the idea of transport of matrix eigenvectors.
+
+```@docs
+cma_es
+Manopt.eigenvector_transport!
+```
+
+## State
+
+```@docs
+CMAESState
+```
+
+## Stopping Criteria
+
+```@docs
+CMAESConditionCov
+StopWhenBestCostInGenerationConstant
+StopWhenEvolutionStagnates
+StopWhenPopulationCostConcentrated
+StopWhenPopulationDiverges
+StopWhenPopulationStuckConcentrated
```

Comment:
It's positive but here it not only means that population is concentrated but also that it's unlikely it will get de-concentrated. So "stuck" was supposed to refer to the latter part of the condition. Maybe `StopWhenPopulationStronglyConcentrated`?

Reply:
Yes that does sound better and is a positive phrasing that I was looking for.

Stuck sounds a bit more like “they do not move, but they actually would like to”. Stronlgy is as vague as Stuck, but much more positive and also emphasises that it is “more” that just concentrated. So let‘s go with that.